.github/CODEOWNERS

```
@@ -601,6 +601,7 @@ pep-0719.rst  @Yhg1s
 pep-0720.rst  @FFY00
 pep-0721.rst  @encukou
 pep-0722.rst  @pfmoore
+pep-0723.rst  @ofek
```

This must be the PEP sponsor -- see my comment above. You can't be listed in CODEOWNERS as you're not a core developer or PEP editor.

```suggestion
pep-0723.rst
```

---

I didn't know, thanks! What should I put here? The PEP delegate?

---

See [PEP 1](https://peps.python.org/pep-0001/#submitting-a-pep) for the details. You need to find someone to be a Sponsor for the PEP -- I'm happy to, or you could ask another core developer or PEP Editor.

---

I added you, thank you very much for volunteering!